doc/examples/Format_Features/Format_Features.ipynb

```diff
@@ -55,7 +55,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "For the sake of brevity, simulation and plotting is done in a separate function. Details on these steps can be found on [other notebooks](../Lotka_Volterra_python/Lotka_Volterra.ipynb)."
+    "For the sake of brevity, simulation and plotting is done in a separate function. Details on these steps can be found on [other notebooks](https://github.com/yaml2sbml-dev/yaml2sbml/tree/main/doc/examples/Lotka_Volterra)."
```

Comment:
here I guess one has to decide for rtd or github, github should be fine though.

Reply:
```suggestion
    "For the sake of brevity, simulation and plotting is done in a separate function. Details on these steps can be found in [other notebooks](https://github.com/yaml2sbml-dev/yaml2sbml/tree/main/doc/examples/Lotka_Volterra)."
```